tutor/src/screens/task/step/exercise-free-response.js

```diff
@@ -184,7 +184,7 @@ class FreeResponseInput extends React.Component {
                       <RevertButton size="lg" ux={ux} />
                       <AnswerButton
                           size="lg"
-                          data-test-id="submit-answer-btn"
+                          data-test-id={`${ux.isSubmitDisabled ? 'submit-answer-btn-disabled' : 'submit-answer-btn'}`}
```

Comment:
hm, I see what you're attempting here, but it seems like it's changing the code a bit too much to make it easier to test.  Instead how about testing that the input is actually disabled like this:
```
expect(
    await page.isDisabled('testEl=submit-answer-btn')
).toBeTruthy()
```

https://playwright.dev/docs/api/class-page#pageisdisabledselector-options

Reply:
@nathanstitt I wanted to do that, but somehow `page.isDisabled` is not available. Maybe we have an old version of Playwright or Puppeteer?